src/pat-display-time.js

```diff
@@ -1,58 +1,59 @@
 (function (root, factory) {
-    if (typeof define === 'function' && define.amd) {
+    // We use AMD (Asynchronous Module Definition) or browser globals to create
+    // this module.
+    if (typeof define === "function" && define.amd) {
         define([
-            'pat-registry',
-            'pat-parser',
-            'moment'
-            ], function() {
-                return factory.apply(this, arguments);
-            });
-        } else {
-            factory(root.patterns, root.patterns.Parser);
-        }
-}(this, function(registry, Parser, moment) {
-    'use strict';
+            "jquery",
+            "pat-base",
+            "pat-registry",
+            "pat-parser",
+            "pat-logger",
+            "moment"
+        ], function() {
+            return factory.apply(this, arguments);
+        });
+    } else {
+        // If require.js is not available, you'll need to make sure that these
+        // global variables are available.
+        factory($, patterns.Base, patterns, patterns.Parser, patterns.logger);
+    }
```

Comment:
moment needs to be passed in here, like so:

`factory($, patterns.Base, patterns, patterns.Parser, patterns.logger, moment);`